src/devnet-utils.ts

```diff
@@ -109,4 +109,22 @@ export class DevnetUtils implements Devnet {
             return response.data;
         }, "Request failed. Make sure your network has the /set_time endpoint");
     }
+
+    public async dump(path: string) {
+        return this.withErrorHandler<void>(async () => {
+            await axios.post(`${this.endpoint}/dump`, {
+                path
+            });
+        }, "Request failed. Make sure your network has the /dump endpoint");
+    }
+
+    public async load(path: string) {
+        return this.withErrorHandler<void>(async () => {
```

Comment:
did you run prettier on this file?

Reply:
Yes, checked it again!

Reply:
https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/134/commits/8ae230449147e8069d6b649c693f9f7614b0e30d Fixed it with a new line before body.